packages/stark-build/config/webpack-partial.common.js

```diff
@@ -65,22 +65,19 @@ module.exports = metadata => {
 		/**
 		 * Options affecting the normal modules.
 		 *
-		 * See: http://webpack.github.io/docs/configuration.html#module
+		 * See: https://webpack.js.org/configuration/module
 		 */
 		module: {
 			rules: [
-				// TsLint loader support for *.ts files
-				// reference: https://github.com/wbuchwalter/tslint-loader
```

Comment:
Can we keep these 2 lines at least? So we have a reference on where to find the docs

Reply:
Updated